docs/contributing/code-style.md

```diff
@@ -292,10 +292,6 @@ type changes in the future.
 -   For string formatting, use `x % (y,)` rather than `x % y`, to avoid
     ambiguity if `y` happens to be a tuple.
 
-### Tests
-
-All significant new features should come with tests. See testing.
-
```

Comment:
I think we should have a section on tests here, but I agree this was was useless.  I wrote a quick replacement that I'll include when I merge this.